src/JabbR-Core/Properties/Chat.dictionary.css

```diff
@@ -0,0 +1,54 @@
+﻿.dictionary_wrapper
```

Comment:
This was all previously under content/themes/default, lets keep out of properties (how did it end up here?)